### PR TITLE
Improve startup time

### PR DESCRIPTION
**Pull request by zeha**
This, on top of #21, takes us from an average of 2.5sec on a fully booted arm64 small ISO to 0.5sec.

### CODE REVIEW

**Comment by zeha** [comment]
jftr, this seemed bad:

plot

**Comment by zeha** [comment]
Better:

plot-new

(This uses a hack to optimise away /etc/zsh/zshenv; also need to think about that file...)

**Comment by zeha** [comment]
In my arm64 VM boot time is noticeably better. Unfortunately as a side-effect, grml-quickconfig has a higher change of not showing the IPv4 address from DHCP :-(

File: bin/save-config [hunk 1]
```diff
@@ -18,21 +18,17 @@
...
   [[ $UID != 0 ]] && runas='sudo' # important for /etc
 
-  if [ -d /run/live/overlay/rw ] ; then # since Dec 2018
+  if [ -d /run/live/overlay/rw ] ; then
     CHANGE_DIR='/run/live/overlay/rw'
-  elif [ -d /lib/live/mount/overlay/rw ] ; then # 2012 until Dec 2018, backwards compatibility
-    CHANGE_DIR='/lib/live/mount/overlay/rw'
   else
-    echo "Error: no overlay directories found (like /lib/live/mount/overlay or /live/overlay)." >&2
+    echo "Error: no overlay directories found (expected /lib/live/mount/overlay)." >&2
```
Comment (mika) [inline bin/save-config:24]: that should report /run/live/overlay?

**Review by mika — COMMENTED** [review]
Wonderful, only one minor issue to clarify

File: bin/save-config (hunk 1 above)
Comment (zeha) [reply bin/save-config]: indeed! Fixed

**Review by mika — APPROVED** [review]
This is *great*, thx!

**Pull request merged**